beacon_node/network/src/network_beacon_processor/gossip_methods.rs

```diff
@@ -1444,6 +1444,12 @@ impl<T: BeaconChainTypes> NetworkBeaconProcessor<T> {
             }
         }
 
+        // Block is gossip valid. Attempt to fetch blobs from the EL using versioned hashes derived
+        // from kzg commitments, without having to wait for all blobs to be sent from the peers.
+        let publish_blobs = true;
+        self.fetch_engine_blobs_and_publish(block.clone(), block_root, publish_blobs)
+            .await;
```

Comment:
@jimmygchen points out that we probably shouldn't await this here as it will delay block processing. We could spawn a task and do it in parallel.